openslides_backend/action/motion/update.py

```diff
@@ -15,13 +16,19 @@ class MotionUpdate(UpdateAction):
 
     model = Motion()
     schema = DefaultSchema(Motion()).get_update_schema(
-        optional_properties=["title", "statute_paragraph_id"]
-    )  # TODO number, modified_final_version, reason, text, amendmend_paragraphs, lead_motion_id, attachment_ids
-
-    def prepare_dataset(self, payload: ActionPayload) -> DataSet:
-        for instance in payload:
-            instance["last_modified"] = round(time.time())
-        return super().prepare_dataset(payload)
+        optional_properties=[
+            "title",
+            "number",
+            "text",
+            "reason",
+            "amendment_paragraph_",
```

Comment:
This is wrong. It is a template field so you can not write directly into "amendment_paragraph_"